tests/manual/version-collision.html

```diff
@@ -0,0 +1,7 @@
+<script>
+	window.CKEDITOR_VERSION = 'the.colliding.version';
```

Comment:
I would make this test more realistic. For instance, load 2 editor builds.

Reply:
You can't do that in a manual test.

Reply:
Ok, that's fine for me, then.